test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -773,6 +773,42 @@ Describe "Hard link and symbolic link tests" -Tags "CI", "RequireAdminOnWindows"
             $childB.Name | Should -BeExactly $childA.Name
         }
     }
+
+    Context "Show immediate target" {
+        BeforeAll {
+            $testDir = Join-Path $TestDrive "immediate-target"
+            New-Item -ItemType Directory $testDir > $null
+
+            $testFile = Join-Path $testDir "target"
+            Set-Content -Path $testFile -Value "Hello world"
+
+            Push-Location $testDir
+            New-Item -ItemType SymbolicLink -Path firstLink -Value target > $null
+            New-Item -ItemType SymbolicLink -Path secondLink -Value firstLink > $null
+            Pop-Location
+        }
+
+        AfterAll {
+            Remove-Item $testDir -Recurse -Force
+        }
+
+        It "Property 'Target' should show the immediate target" {
+            $firstLink = Get-Item (Join-Path $testDir 'firstLink')
+            $firstLink.Target | Should -BeExactly 'target'
+            $str = [Microsoft.PowerShell.Commands.FileSystemProvider]::NameString($firstLink)
+            [System.Management.Automation.Internal.StringDecorated]::new($str).ToString([System.Management.Automation.OutputRendering]::PlainText) | Should -BeExactly 'firstLink -> target'
```

Comment:
Could you please use `$PSStyle.OutputRendering = 'PlainText'`?

Reply:
`$PSStyle.OutputRendering` doesn't affect `FileSystemProvider.NameString`. Please take a look at `FileSystemProvider.NameString` method for details.

Reply:
Nit: should it be fixed?

Reply:
I don't think there is anything to be fixed here. `NameString` is a utility method, and whether you call it or not from formatting code depends on `$PSStyle.OutputRenderin`.